sui/its.js

```diff
@@ -501,27 +505,41 @@ async function linkCoin(keypair, client, config, contracts, args, options) {
         symbol,
         metadata,
         tokenType,
+        tokenManager === tokenManagerTypes[1] ? treasuryCap : null,
```

Comment:
```suggestion
        tokenManagerType === MINT_BURN ? treasuryCap : null,
```
weird to rely on the index of an array for comparison. if that gets changed, this will break. Define token manager type constants and compare against those (and use it in the CLI arg choices)